Pod/Classes/Base/MediaControl.swift

```diff
@@ -143,7 +165,25 @@ public class MediaControl: UIBaseObject {
     }
     
     private func containerReady() {
+        livePlayback = container.playback.type() == .Live
+        livePlayback ? setupForLive() : setupForVOD()
+        updateBars()
+        updateScrubberPosition()
+        updatePlaybackControlButtonIcon()
+    }
+    
+    private func setupForLive() {
+        seekPercentage = 1
+        progressBarView.backgroundColor = UIColor.redColor()
+        durationLabel.text = ""
+        currentTimeLabel.text = ""
```

Comment:
@barbosa 

Actually I was not hiding it. It's a discreet separator but it does look odd when paying attention